opentelemetry-api/src/opentelemetry/context/__init__.py

```diff
@@ -150,3 +152,20 @@ def __repr__(self):
     from .thread_local_context import ThreadLocalRuntimeContext
 
     Context = ThreadLocalRuntimeContext()
+
+
+class ContextAPI:
```

Comment:
See https://github.com/open-telemetry/opentelemetry-python/pull/379/files#r371754360. We might want to stay closer to contextvars.

Reply:
Yes, that's right. This change only implements the stub context API as a way to break apart https://github.com/open-telemetry/opentelemetry-python/pull/325 to make it easier to review. The next change in this branch will be the Context API changes, followed by the Propagation API.

Reply:
One consideration that might affect the design here: do we need the context API to support arbitrary "concerns", or is it safe to assume adding a new concern will require a code change here? If the former, we probably want a single dict-valued `ContextVar` with a (probably also dict-valued) key for each concern. If the latter, we could have one cv for trace context and another for correlation context.